modules/contact/doc/content/modules/contact/index.md

```diff
@@ -271,24 +271,37 @@ The recommended PETSc options for use with `NodeFaceConstraint` based contact ar
 
 !syntax complete groups=ContactApp level=3
 
-## Automatic scaling
+## Scaling effects
 
-Automatic scaling can significantly improve both linear and non-linear
-convergence for contact problems. The below linear and non-linear convergence
-statistics are based on running
+The effects of scaling on non-linear solve convergence are shown below for
+different kinds of contact formulations. The results are based on the input files for
 [RANFS](/ranfs-and-scaling/bouncing-block-ranfs.i),
 [kinematic](/bouncing-block-kinematic.i), and
-[tangential penalty](/bouncing-block-tan-pen.i) test input files with two processes.
-
-| Scheme | Cumulative nonlinear iterations | Cumulative linear iterations |
-| --- | --- | --- |
-| RANFS no scaling AMG | 66 | 397 |
-| RANFS auto-scaling AMG | 56 | 218 |
-| Kinematic no scaling AMG | No convergence | No convergence |
-| Kinematic auto-scaling AMG | 53 | 203 |
-| Kinematic no scaling LU | 65 | 271 |
-| Kinematic auto-scaling LU | 41 | 114 |
-| Tangential Penalty no scaling AMG | No convergence | No convergence |
-| Tangential Penalty auto-scaling AMG | 53 | 208 |
-| Tangential Penalty no scaling LU | 60 | 224 |
-| Tangential Penalty auto-scaling LU | 41 | 116 |
+[tangential penalty](/bouncing-block-tan-pen.i),
+running with two processes.
+
+| Scheme | Cumulative nonlinear iterations | Cumulative linear iterations |  Condition number | Scaling Factor |
+| - | - | - | - | - |
```

Comment:
@lindsayad I bet this fixed it, I can get it to fail.

Reply:
I actually had a row that had two many columns...sorry I should have notified you that I fixed it